lib/ts/supertokens.ts

```diff
@@ -49,6 +50,9 @@ export default class SuperTokens {
     telemetryEnabled: boolean;
 
     constructor(config: TypeInput) {
+        if (config.debug) {
+            debug.enable("com.supertokens");
```

Comment:
- bad idea to hard code this when we already have `SUPERTOKENS_DEBUG_NAMESPACE` in the logger.ts file.
- Ideally this boolean is passed to the logger.ts file and that then enables the debug logging.. we do not want to directly interact with the `debug` lib in this file.